README.rst

```diff
@@ -149,6 +149,10 @@ On Windows::
 (If that failed, try::
     
     $ pip install pypiwin32
+ 
+ then grab a proper version of pypiwin32 installer and install it::
+ 
+    https://github.com/mhammond/pywin32/releases
```

Comment:
Yeah, let's completely get rid of references to pypiwin32 and simplify. How about something like this?

`On Windows:

Install pywin32 from here:

https://github.com/mhammond/pywin32/releases`